zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/RequestMapperTest.java

```diff
@@ -57,6 +64,20 @@ void tokenContainsUsernameWithBasicAuth() {
 
   @Test
   void tokenContainsExtraClaimsWithOidcAuth() {
```

Comment:
:wrench: The test name actually points us to missing test coverage: as is, this is only testing that the username is extracted. We should also assert that other claims are still forwarded, just like with normal JWTs.